### DISCUSSION TITLE
Why does the Accordion remove `Accordion.Content` from the DOM?

### DISCUSSION THREAD

**Discussion by JeffBeltran** [radix-ui/primitives#3244] [Help]
We noticed that our FAQ content was not being rendered in the DOM when the accordion was collapsed, and we’re concerned about potential SEO implications. Upon reviewing the [W3 ARIA accordion example](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion), it seems removing the content from the DOM isn’t an accessibility requirement. In fact, the example provided by W3C simply hides the content using CSS. Additionally, I noticed that libraries like Headless UI also follow this CSS-based approach.

Could you clarify the rationale behind this implementation in Radix UI? Are there specific considerations (e.g., performance, accessibility) that led to this decision? If SEO or accessibility might be impacted, are there recommended workarounds for ensuring hidden content is still discoverable by search engines?

**Comment by tranhoanvo** [comment]
Any updates on this ?
